packages/fxa-content-server/app/styles/modules/_branding.scss

```diff
@@ -36,8 +36,20 @@
   }
 }
 
-#main-content.send-sms::before {
-  content: initial;
+// Note this is non-standard styling for #main-content. It would
+// generally have a top padding of 70px desktop / 82px mobile
```

Comment:
Small nit: these values can change pretty easily and the comment would then be out-of-date, something like "additional" or "more" padding would be a little more evergreen. (Or even just "Less padding is needed when the Firefox logo is hidden" is fine)